hack/build-local-images.py

```diff
@@ -227,4 +236,4 @@ def debug(message):
     print "[ERROR] This script knows how to build:\n\t{}".format(
         "\n\t".join(map(full_name, image_config.keys()))
     )
-    exit(1)
\ No newline at end of file
+    exit(1)
```

Comment:
yes that's not the whitespace i was worried about.  it's the stuff above.  which, upon closer analysis *appears* to be making it consistent w/ the rest of the file?